cpu/efm32/periph/uart.c

```diff
@@ -131,15 +129,16 @@ int uart_init(uart_t dev, uint32_t baudrate, uart_rx_cb_t rx_cb, void *arg)
 
         /* enable receive interrupt */
         LEUART_IntEnable(leuart, LEUART_IEN_RXDATAV);
```

Comment:
I guess this also needs to be wrapped inside an if-statement

Reply:
What I did is leave the interrupt always enabled in the peripheral and put only an if-block around enabling the interrupt in the NVIC. I don't think there would be a functional change but I'm mostly indifferent about it and I'll add if-blocks if you disagree.